lua/cfc_time/server/time_tracking.lua

```diff
@@ -57,29 +58,23 @@ function ctime:updateTimes()
     local timeDelta = now - self.lastUpdate
 
     for steamID, data in pairs( self.sessions ) do
-        local isValid = true
-
         local joined = data.joined
         local departed = data.departed
 
         if departed and departed < self.lastUpdate then
             self:untrackPlayer( steamID )
-            isValid = false
-        end
-
-        local sessionTime = ( departed or now ) - joined
-        if sessionTime <= 0 then
-            isValid = false
-        end
-
-        if isValid then
+        else
+            local sessionTime = ( departed or now ) - joined
             data.duration = sessionTime
 
             local sessionID = self.sessionIDs[steamID]
             batch[sessionID] = data
 
-            local newTotal = self.totalTimes[steamID] + timeDelta
-            self.totalTimes[steamID] = newTotal
+            local currentTotal = self.totalTimes[steamID]
+            if currentTotal then
+                local newTotal = currentTotal + timeDelta
+                self.totalTimes[steamID] = newTotal
+            end
         end
```

Comment:
Summary:

If the player is departed or the session time is 0, it should still be updated to the DB.

However, if the player doesn't have a totalTime, then we shouldn't update/set their total time.
This should prevent NULL entities in the total times (so it won't try to SetNW on them)